packs/ac-feats/Billowing_Wings_p83NzvyOMMa5Dgdh.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "Billowing Wings",
+  "type": "feat",
+  "effects": [],
+  "system": {
+    "description": {
+      "gm": "",
+      "value": "<p>You’ve trained your winged companions to gather air in their wings, whipping up a concentrated gust of wind at opponents, potentially staggering them.</p><p>Your companions with wings gain a gust ranged unarmed attack, which deals @Damage[1d4[bludgeoning]] damage with a range of 30 feet and has the air and propulsive traits. On a critical hit, the target is pushed back 5 feet. This is forced movement.</p>"
+    },
+    "rules": [
+      {
+        "key": "Strike",
+        "category": "unarmed",
+        "damage": {
+          "base": {
+            "damageType": "bludgeoning",
+            "dice": 1,
+            "die": "d4"
+          }
+        },
+        "group": "brawling",
+        "label": "Gust",
+        "traits": [
+          "air",
+          "propulsive",
+          "unarmed"
+        ],
+        "range": {
+          "max": 30
+        },
+        "slug": "gust",
+        "img": "systems/pf2e/icons/unarmed-attacks/gust.webp"
+      },
+      {
+        "key": "Note",
+        "selector": "gust-attack-roll",
+        "text": "On a critical hit, the target is pushed back 5 feet. This is forced movement.",
+        "title": "Gust",
+        "outcome": [
+          "criticalSuccess"
+        ]
+      }
+    ],
+    "slug": null,
+    "_migration": {
+      "version": 0.935,
+      "lastMigration": null
+    },
+    "traits": {
+      "otherTags": [],
+      "value": [],
+      "rarity": "common"
+    },
+    "publication": {
+      "title": "Howl of the Wild",
+      "authors": "",
+      "license": "ORC",
+      "remaster": true
+    },
+    "level": {
+      "value": 12
+    },
+    "category": "ancestryfeature",
+    "onlyLevel1": false,
+    "maxTakable": 1,
+    "actionType": {
+      "value": "passive"
+    },
+    "actions": {
+      "value": null
+    },
+    "prerequisites": {
+      "value": []
+    },
+    "location": null
+  },
+  "_id": "p83NzvyOMMa5Dgdh",
+  "img": "systems/pf2e/icons/spells/gust-of-wind.webp",
```

Comment:
I'll swap the icons later, but they should all be `icons/sundries/books/book-red-exclamation.webp` since these are feats